internal/compiler/lookup.rs

```diff
@@ -43,6 +50,8 @@ pub struct LookupCtx<'a> {
 
     /// The token currently processed
     pub current_token: Option<NodeOrToken>,
+
+    pub phase: LookupPhase,
```

Comment:
I thought about this, And i'd prefer this would be a separate argument to `from_qualified_name_node` rather than in the LookupCtx.

Reply:
Ok, can do